src/System.Private.ServiceModel/tools/scripts/StartWCFService.cmd

```diff
@@ -39,12 +39,14 @@ if NOT [%ERRORLEVEL%]==[0] (
     echo Warning: An error occurred while configuration https port.
   )
 
+REM
 REM Start the self hosted WCF Test Service
-call %~dp0..\..\..\..\bin\Wcf\tools\SelfHostedWcfService\SelfHostedWcfService.exe %*
+REM
+call %~dp0..\..\..\..\bin\Wcf\tools\SelfHostedWcfService\SelfHostedWcfService.exe
```

Comment:
Is pausing (i.e. blocking execution) at this point the right thing to do?  The EnsureBridgeRunning used to launch a new process.  I found it awkward to have to open a 2nd window for this.